includes/wikia/services/WikiDetailsService.class.php

```diff
@@ -190,7 +192,7 @@ protected function getFromService( $id ) {
 			}
 		}
 
-		return [
+		$stats = [
 			'stats' => [
```

Comment:
Fields like `headline` or `lang` are not stats ;) `$wikiDetails` would be more appropriate.